### ISSUE TITLE
Luminosity ratio for 'Expand' icon button is 2.4:1 which is less than 3:1

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1131]
A team member will update this issue with more details soon.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.

**Comment by varunv997** [comment]
Hi I'd like to work on this.
I am trying to reproduce the problem. However, I am not able to find "Graphing" Tab in the Menu. Can somebody help? Thanks 
![image](https://user-images.githubusercontent.com/18118137/79636683-1fb19d00-8197-11ea-9588-9a8b92d5a84a.png)

**Comment by sanderl** [comment]
> Hi I'd like to work on this.
> I am trying to reproduce the problem. However, I am not able to find "Graphing" Tab in the Menu. Can somebody help? Thanks
> ![image](https://user-images.githubusercontent.com/18118137/79636683-1fb19d00-8197-11ea-9588-9a8b92d5a84a.png)

Hi varunv997,

Graphing mode is only enabled on the latest Windows Insider builds at this time. Once you have one of those newer builds installed, you should be able to see the Graphing option. Please let us know if you are still having problems after you are on a newer build.
